frontend/public/components/start-guide.tsx

```diff
@@ -12,39 +13,46 @@ import { ProjectModel } from '../models';
 import { createProjectModal } from './modals/create-namespace-modal';
 
 export const OpenShiftGettingStarted = connect(createProjectMessageStateToProps)(
-  ({ canCreateProject = true, createProjectMessage }: OpenShiftGettingStartedProps) => (
-    <>
-      {canCreateProject ? (
+  ({ canCreateProject = true, createProjectMessage }: OpenShiftGettingStartedProps) => {
+    const { t } = useTranslation();
+    return (
+      <>
+        {canCreateProject ? (
+          <p>
+            {t(
+              'public~OpenShift helps you quickly develop, host, and scale applications. To get started, create a project for your application.',
+            )}
+          </p>
+        ) : (
+          <p>
+            {t(
+              'public~OpenShift helps you quickly develop, host, and scale applications. To get started, you&lsquo;ll need a project. Currently, you can&lsquo;t create or access any projects.',
+            )}
+            {!createProjectMessage &&
+              t('public~ You&lsquo;ll need to contact a cluster administrator for help.')}
```

Comment:
Won't the HTML entity show up incorrectly in the message?

We should add the leading space outside of the message.

Reply:
replaced 's and add &nbsp; spacing similar to other strings